client/plots/DEanalysis.js

```diff
@@ -53,6 +53,25 @@ class DEanalysis {
 	async setControls(output) {
 		this.dom.controlsDiv.selectAll('*').remove()
 		const inputs = [
+			{
+				label: 'Min count',
```

Comment:
for these two new params, will it be better to call them `Min read count per sample` and `Min total read count`
at a minimum, if the cutoff is about read count, then label must indicate so

Reply:
Discussed privately in slack